draft-jmiller-json-proof-token.md

```diff
@@ -0,0 +1,148 @@
+
+{mainmatter}
+
+# Introduction
+
+JSON Proof Token (JPT) is a compact claims representation format intended to be used in the same ways as a JWT but with additional support for selective disclosure and unlinkability.  JPTs encode claim values to be transmitted as JSON payloads of a JSON Web Proof (JWP).  JPTs are always represented using the JWP Compact Serialization.
+
+# Conventions and Definitions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in BCP 14 [@!RFC2119] [@RFC8174]
+when, and only when, they appear in all capitals, as shown here.
+
+# Background
+
+JWP defines a clean abstraction around the parts of a container binding together a protected header and one or more payloads.  It does not define how claims are organized into payloads and what formats they are in.  JPTs are intended to be as close to a JWT as possible, while also supporting the selective disclosure and unlinkability of JWPs.
+
+# Design Considerations
+
+The rationale behind the deisgn for JSON Proof Tokens is important when considering how it is structured.  These sections detail the underlying reasoning for the approach defined by JPTs.
+
+## Unlinkability
+
+Supporting unlinkability is perhaps the most challenging design constraint for JPTs.  Even the smallest oversight can introduce a subtle vector for relying parties to collude and correlate one or more subjects across their usage.
+
+The principle tools to prevent this are data minimization and uniformity.  All data included in an individual JPT must be critically minimized to only the values that are cryptographic in nature and able to be selectively disclosed with consent or transformed by the proof algorithm when presented.
+
+Any other data that is repeated across multiple JPTs must be externalized so that it is uniform across every issuance.  This includes preventing the usage of optional headers, dynamic mapping of claims to payloads, changes to how many payloads are included, and the ordering of the payloads.
+
+## Selective Disclosure
+
+While JWPs provide the underling structure for easily supporting selective disclosure, JPTs must go a step further to ensure that applications can effectively provide choice and consent on exactly what is being disclosed.
+
+To accomplish this it is important that every single payload that is disclosed is understood by the supporting software as to which claim it is and what that value is.  JPTs do not support disclosing claims that are intended to be private from the issuer to the relying party, all revealed payloads must be mapped to a claim that is accessible to the application.
```

Comment:
Per the discussion on the 2-Nov-21 DIF JWP call, we need clearer language than "understood".  There's a distinction between knowing how to disclose a claim as a mechanical process and full understanding the semantics of a claim.